tools/c7n_oci/c7n_oci/handler.py

```diff
@@ -0,0 +1,84 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import io
+import json
+import logging
+import os
+import uuid
+
+import yaml
+from c7n_oci.provider import OCI
+
+from c7n.config import Config
+from c7n.policy import PolicyCollection
+from c7n.resources import load_resources
+from c7n.structure import StructureParser
+from c7n.utils import reset_session_cache
+
+log = logging.getLogger("custodian.oci.functions")
+logging.getLogger().setLevel(logging.INFO)
+
+
+def handler(ctx, data: io.BytesIO = None):
+    log.info("Starting Function execution")
+    try:
+        body = json.loads(data.getvalue())
+        log.debug(f"Recieved Body {str(body)}")
+    except (Exception, ValueError) as ex:
+        log.error("Error parsing json payload: ")
+        log.exception(ex)
+        raise ex
+
+    run(body, ctx)
+    log.info("Function execution is completed")
+
+
+def run(event, ctx):
+    function_config = ctx.Config()
```

Comment:
so ctx, is https://github.com/fnproject/fdk-python/blob/master/fdk/context.py

but where does Config() dict get populated from?

Reply:
Config() returns the config attribute, which is being set to os.environ at : https://github.com/fnproject/fdk-python/blob/c6ec8d66cd58889c0031edd0b7fc3ae5c836926d/fdk/context.py#L334.